napari/_qt/menus/file_menu.py

```diff
@@ -19,6 +19,7 @@ class FileMenu(QMenu):
     def __init__(self, window: 'Window'):
         self._win = window
         super().__init__(trans._('&File'), window._qt_window)
+        self.aboutToShow.connect(self._update)
```

Comment:
Should we have a `NapariMenu` that implements this about to show and update connection by default @tlambert03